djstripe/stripe_objects.py

```diff
@@ -68,12 +57,15 @@ class StripeObject(TimeStampedModel):
     objects = models.Manager()
     stripe_objects = StripeObjectManager()
 
+    stripe_id = StripeIdField(unique=True, stripe_name='id')
+    livemode = StripeNullBooleanField(default=False, null=True,
+                                      help_text="Null here indicates that data was unavailable. Otherwise, this field "
+                                                "indicates whether this record comes from Stripe test mode or live "
+                                                "mode operation.")
+
```

Comment:
Gotcha. Is that the only case you can forsee?

"data was unavailable" to me meant that the model's data was unavailable, not just the livemode field data. Maybe change it to "Null here indicates that the livemode status is unknown or was previously unrecorded."